main.rb

```diff
@@ -0,0 +1,64 @@
+require 'json'
+require 'date'
+require './classes/item'
+require './classes/genre'
+require './classes/music_album'
+require './classes/book'
+require './classes/label'
+require './classes/author'
+require './classes/game'
+require './operations/book_operations'
+require './operations/music_album_operations'
+require './operations/game_operations'
+require './operations/file_operations'
+require './operations/item_operations'
+
+items = load_items_from_json
+
+puts 'Welcome to the Catalogue of my things'
+
+def print_options
+  puts "\nOptions"
+  puts '1. List all books'
+  puts '2. List all music albums'
+  puts '3. List all games'
+  puts '4. List all genres'
+  puts '5. List all labels'
+  puts '6. List all authors'
+  puts '7. Add a book'
+  puts '8. Add a music album'
+  puts '9. Add a game'
+  puts '10. Quit'
+end
```

Comment:
- [ ] Great work making most of the required changes, however, I noticed that when I try to _**Add a book**_ or try to _**Add a game**_, I get errors in the console application that you created. Kindly fix these errors. 👍 😎 

#### Error When trying to add a book

![image](https://github.com/brighton0102/Catalog-of-my-things/assets/77106962/4efd248b-b0b1-4073-8b53-af38f88fc72a)

#### Error when trying to add a game

![image](https://github.com/brighton0102/Catalog-of-my-things/assets/77106962/2e1ec206-16da-41af-a225-9716d4db8d0f)

#### N.B: Please ensure that all options of the console application work properly after fixing the errors and that a user can save data to JSON files and load data fro the JSON files upon reentry. 💻 🖱️